manifests/role/argo-rollouts-clusterrole.yaml

```diff
@@ -181,3 +181,14 @@ rules:
   - update
   - list
   - delete
+- apiGroups:
+  - route.openshift.io
+  resources:
+  - routes
+  verbs:
+  - create
+  - watch
+  - get
+  - update
+  - list
+  - delete
```

Comment:
I don't think we should give create or delete to this, since we are only modifying an existing one.